# HG changeset patch
# User Rik <rik@octave.org>
# Date 1455859201 28800
#      Thu Feb 18 21:20:01 2016 -0800
# Node ID 0152d8745c3dc95c4c2679d8b25894c6879279e5
# Parent  46284491ffc5dbee3b913e38ceacd2ddcadeef9b
build: User serial comma in warnings for sndfile and portaudio.

* configure.ac: User serial comma in warnings for sndfile and portaudio.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -1004,24 +1004,24 @@ if test -z "$warn_curl"; then
       [Define to the legacy option name if using an older version of cURL.])
   fi
 fi
 LIBS="$save_LIBS"
 CPPFLAGS="$save_CPPFLAGS"
 
 ### Check for sndfile
 
-warn_sndfile="sndfile library fails tests.  The audioinfo, audioread and audiowrite functions for reading and writing audio files will not be fully functional."
+warn_sndfile="sndfile library fails tests.  The audioinfo, audioread, and audiowrite functions for reading and writing audio files will not be fully functional."
 
 check_sndfile=no
 AC_ARG_WITH([sndfile],
   [AS_HELP_STRING([--without-sndfile],
     [don't use sndfile library, disable audio file I/O])],
   [if test x"$withval" = x"no"; then
-     warn_sndfile="--without-sndfile specified.  The audioinfo, audioread and audiowrite functions for reading and writing audio files will not be fully functional."
+     warn_sndfile="--without-sndfile specified.  The audioinfo, audioread, and audiowrite functions for reading and writing audio files will not be fully functional."
    else
      check_sndfile=yes
    fi],
   [check_sndfile=yes])
 
 if test $check_sndfile = yes; then
   PKG_CHECK_EXISTS([sndfile], [
     SNDFILE_CPPFLAGS=`$PKG_CONFIG --cflags-only-I sndfile`
@@ -1041,24 +1041,24 @@ else
 fi
 
 AC_SUBST(SNDFILE_CPPFLAGS)
 AC_SUBST(SNDFILE_LDFLAGS)
 AC_SUBST(SNDFILE_LIBS)
 
 ### Check for PortAudio
 
-warn_portaudio="PortAudio library fails tests.  The audioplayer, audiorecorder classes and audiodevinfo function for audio playback and recording will not be fully functional."
+warn_portaudio="PortAudio library fails tests.  The audioplayer, audiorecorder classes, and audiodevinfo function for audio playback and recording will not be fully functional."
 
 check_portaudio=no
 AC_ARG_WITH([portaudio],
   [AS_HELP_STRING([--without-portaudio],
     [don't use PortAudio library, disable audio playback and recording])],
   [if test x"$withval" = x"no"; then
-     warn_portaudio="--without-portaudio specified.  The audioplayer, audiorecorder classes and audiodevinfo function for audio playback and recording will not be fully functional."
+     warn_portaudio="--without-portaudio specified.  The audioplayer, audiorecorder classes, and audiodevinfo function for audio playback and recording will not be fully functional."
    else
      check_portaudio=yes
    fi],
   [check_portaudio=yes])
 
 if test $check_portaudio = yes; then
   PKG_CHECK_EXISTS([portaudio-2.0 >= 19], [
     PORTAUDIO_CPPFLAGS=`$PKG_CONFIG --cflags-only-I portaudio-2.0`
